scripts/check-broken-links.mjs

```diff
@@ -3,17 +3,60 @@ import { LinkChecker } from 'linkinator'
 import chalk from 'chalk'
 import {
   getSitemapLocations,
-  splitPagesIntoSegments,
+  splitPagesIntoBatches,
+  getPagesSlice,
 } from '../playwright/utils/getSitemapLocations.js'
 import fs from 'fs'
 
 const OPTIONS = {
   sitemapUrl: process.env.SITE_URL || 'http://www.va.gov/',
-  batchSize: process.env.BATCH_SIZE || 32,
+  // totalInstances is the number of instances to split work among.
+  totalInstances: process.env.TOTAL_INSTANCES || 1,
```

Comment:
The default total instances and instance number is 1 each. This means that if the vars aren't set, the broken link script will act as before, checking the entire script.

This mechanism can also be (ab)used to do a slice during testing. 64 instances  = ~330 pages per slice. If you set 64 total instances and whatever number between 1 and 64 for the instance, you'll get a slice of the full scan.